k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageViewFragment.java

```diff
@@ -165,7 +166,7 @@ public void onDestroy() {
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
-            Bundle savedInstanceState) {
+                             Bundle savedInstanceState) {
```

Comment:
Wrong indentation. Import our code style settings to stop the IDE from making these changes.